src/front/spv/mod.rs

```diff
@@ -1123,7 +1123,7 @@ impl<I: Iterator<Item = u32>> Frontend<I> {
         block: &mut crate::Block,
         block_id: spirv::Word,
         body_idx: usize,
-        axis: crate::DerivativeAxis,
+        (axis, ctrl): (crate::DerivativeAxis, crate::DerivativeControl),
```

Comment:
I got hit with `too many args` from clippy. Not sure if this is a proper solution or not